pym/bob/utils.py

```diff
@@ -741,7 +741,8 @@ def __exit__(self, exc_type, exc_value, traceback):
         asyncio.set_event_loop(None)
 
 
-async def run(args, universal_newlines=False, errors=None, check=False, shell=False, **kwargs):
```

Comment:
This isn't used, is it?

Reply:
It seams to be used by `predictLiveBuildId` ->  `await check_output`   -> `await check_output` -> `run`

Reply:
Aaahhh, of course. Never mind.